docs/tutorials/deployments.md

```diff
@@ -408,7 +449,7 @@ Agent started! Looking for work from queue(s): test...
 
 Remember that:
 
-- We specified the `test` work queue when building the deployment files.
+- We specified the `test` work queue when creating the deployment.
```

Comment:
I think the terms used here are important and have been trying to intentionally use these terms consistently:

- "Create a deployment" is the entire process of configuring ("building") the deployment and creating a deployment object on the API ("applying").
- "Build a deployment" is the configuration step, equivalent to `prefect deployment build`. You are specifying the configuration of a deployment to be applied.
- "Apply a deployment" is uses the configuration to create or updated a deployment object on the API, equivalent to `prefect deployment apply`. You are applying configuration to an object on the server/Cloud database via the API.

We don't have to use this nomenclature, but I found it useful to be careful about aligning the terms to the actions.